File: 30_days_Python_HackerRank_Challenge/operators.py
#python-2
#!/bin/python
import math
import os
import random
import re
import sys

def solve(meal_cost, tip_percent, tax_percent):

    #solve function and calculation
    tip = meal_cost * tip_percent / 100.0
    tax = meal_cost * tax_percent / 100.0
    total = meal_cost + tip + tax
    
    print int(round(total))

if __name__ == '__main__':
    meal_cost = float(raw_input().strip())
    tip_percent = int(raw_input().strip())
    tax_percent = int(raw_input().strip())
    solve(meal_cost, tip_percent, tax_percent)

#pythn-3
#!/bin/python3
import math
import os
import random
import re
import sys

def solve(meal_cost, tip_percent, tax_percent):
    tip = meal_cost * tip_percent / 100.0
    tax = meal_cost * tax_percent / 100.0
    total = meal_cost + tip + tax
    
    print(round(total))

if __name__ == '__main__':
    meal_cost = float(input().strip())
    tip_percent = int(input().strip())
    tax_percent = int(input().strip())
    solve(meal_cost, tip_percent, tax_percent)

File: 30_days_C#_HackerRank_Challenge/operators.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Result
{
    public static void solve(double meal_cost, int tip_percent, int tax_percent)
    {
        double tip = meal_cost * tip_percent / 100;
        double tax = meal_cost * tax_percent / 100;
        
        double total = meal_cost + tip + tax;
        
        Console.WriteLine(Math.Round(total));
    }

}

class Solution
{
    public static void Main(string[] args)
    {
      #solve function and calculation
        double meal_cost = Convert.ToDouble(Console.ReadLine().Trim());
        int tip_percent = Convert.ToInt32(Console.ReadLine().Trim());
        int tax_percent = Convert.ToInt32(Console.ReadLine().Trim());

        Result.solve(meal_cost, tip_percent, tax_percent);
    }
}
